src/main/java/frc/robot/RobotContainer.java

```diff
@@ -153,13 +150,15 @@ private void initCommands() {
     if (CDSSubsystem != null && shooterSubsystem != null) {
       CDSForwardCommand = new CDSForwardCommand(CDSSubsystem);
       CDSReverseCommand = new CDSReverseCommand(CDSSubsystem);
+      // CDSSubsystem.getAllianceColor();
+      CDSSubsystem.senseColor();
```

Comment:
I'm a little confused on the difference between the two, why they would be called while binding buttons, and if they do work how I think, can the color sensor see our payload?
I'll put them back, I moved them because they were originally in a if that called commands, so I figured the if statement was misplaced, and merged the functionality with the pre-existing if with commands

Reply:
Agreed it is a little weird, just thinking to leave it where Main has it so intake team can be the one to manage where it is called

Reply:
That way you aren't on the hook if the build or simulation fails for whatever reason